task-launcher/src/tasks/shared/trials/afcStimulus.ts

```diff
@@ -122,13 +123,12 @@ const getPromptTemplate = (
   stimulusContainerClassList: string[],
 ) => {
   let template = '<div class="lev-stimulus-container">';
-  if (!noAudio) {
-    template += `
-      <button id="${replayButtonHtmlId}" class="replay">
-        ${replayButtonSvg}
-      </button>
-    `;
-  }
+
```

Comment:
I think this function no longer uses the `noAudio` so that can be removed.

Reply:
I forgot about that, thanks!